scripts/sct_analyze_lesion.py

```diff
@@ -27,53 +27,72 @@
-    parser.add_option(name="-s",
-                      type_value="file",
-                      description="Spinal cord centerline or segmentation file, which will be used to correct morphometric measures with cord angle with respect to slice.",
-                      mandatory=False,
-                      example='t2_seg.nii.gz')
-    parser.add_option(name="-i",
-                      type_value="file",
-                      description="Image from which to extract average values within lesions (e.g. T2w or T1w image, texture image).",
-                      mandatory=False,
-                      example='t2.nii.gz')
-    parser.add_option(name="-f",
-                      type_value="str",
-                      description="Path to folder containing the atlas/template registered to the anatomical image.",
-                      mandatory=False,
-                      example="./label")
-    parser.add_option(name="-ofolder",
-                      type_value="folder_creation",
-                      description="Output folder",
-                      mandatory=False,
-                      example='./')
-    parser.add_option(name="-r",
-                      type_value="multiple_choice",
-                      description="Remove temporary files.",
-                      mandatory=False,
-                      default_value='1',
-                      example=['0', '1'])
-    parser.add_option(name="-v",
-                      type_value='multiple_choice',
-                      description="Verbose: 0 = nothing, 1 = classic, 2 = expended",
-                      mandatory=False,
-                      example=['0', '1', '2'],
-                      default_value='1')
+
+    parser = argparse.ArgumentParser(
+        description='Compute statistics on segmented lesions. The function assigns an ID value to each\n'
+                    'lesion (1, 2, 3, etc.) and then outputs morphometric measures for each lesion:\n'
+                    '- volume [mm^3]\n'
+                    '- length [mm]: length along the Superior-Inferior axis\n'
+                    '- max_equivalent_diameter [mm]: maximum diameter of the lesion, when approximating\n'
+                    '                                the lesion as a circle in the axial plane.\n\n'
+                    'N.B. If the proportion of lesion in each region (e.g. WM and GM) does not sum\n'
+                    'up to 100%, it means that the registered template does not fully cover the\n'
+                    'lesion. In that case you might want to check the registration results.',
+        add_help=None,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+    mandatory_arguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory_arguments.add_argument(
+        "-m",
+        help='Binary mask of lesions (lesions are labeled as "1").',
+        metavar='',
```

Comment:
@YangHee-Min please add this field to change this:
~~~
  -m M         Binary mask of lesions (lesions are labeled as "1").
~~~

into this:
~~~
  -m          Binary mask of lesions (lesions are labeled as "1").
~~~